pkg/quay/quay.go

```diff
@@ -96,6 +96,44 @@ func (c *QuayClient) CreateRepository(r RepositoryRequest) (*Repository, error)
 	return data, nil
 }
 
+// IsRepositoryExists checks if the specified image repository exists in quay.
+func (c *QuayClient) IsRepositoryExists(organization, imageRepository string) (bool, error) {
```

Comment:
Consider renaming the function to `DoesRepositoryExist` or `RepositoryExists`.
I would personally prefer `DoesRepositoryExist`.